CHANGELOG.md

```diff
@@ -1,11 +1,20 @@
 # Changelog
 
-## Next Release
+## 20.11.0
 
 ### Features Added
 
 - Interlock/Armory II HSM keystore support
-- Eth2 slashing protection data able to be imported from json file (Interchange format V5)
+- Eth2 slashing protection data able to be exported and imported from json file (Interchange format V5)
+- Eth2 signing API can now return response in json format
+- Genesis validators root validation for eth2 signing requests. The GVR is set on first eth2 signing request.
```

Comment:
Only able to sign for a single Genesis validators root (which is defined by the first received request after creation)

Reply:
done